### DISCUSSION TITLE
Deta Visior doesnt show LOGS

### DISCUSSION THREAD

**Discussion by adebisit** [deta/cloud-docs#741] [Help] [answered]
I have enabled Deta Visor but for some reason, I can't see the logs after multiple requests to the endpoint.
It is stuck on the message: "Make a request to your endpoint to put on the visor..."

Please help.

**Comment by jnsougata** [comment]
Are you sure you are checking right micro in a  right project?

**Comment by adebisit** [comment, marked as answer]
Yes pretty sure. I eventually used the `deta logs --follow` command.
